llama.cpp

```diff
@@ -100,7 +102,7 @@ struct llama_hparams {
     int32_t n_head  = 32;
     int32_t n_layer = 32;
     int32_t n_rot   = 64;
-    int32_t f16     = 1;
+    int32_t f16     = LLAMA_FTYPE_MOSTLY_F16;
```

Comment:
How hard is it to change name of this field? This is a private struct, so should be easy, no?